README.md

```diff
@@ -14,22 +14,27 @@
 - OpenGL (Version >= 3.3)
 
 ## Build Instructions
-### Visual Studio
 
-Once cloned, enter the directory, by doing `cd quartz-engine`, and complete the following commands:
+Once cloned, navigate to the projects root directory and execute the following commands in a terminal.
 
-```
-cmake -H. -BBuild
-cmake --build Build
-```
+  1. `cmake -S. -BBuild`
+  2. `cmake --build Build --target QuartzSandbox`
 
-### Linux, Mac OS X, MSYS
+Now follow the platform specific instructions detailed below.
+
+### Visual Studio
+
+  - Open the generated solution file in the `Build/` folder in Visual Studio
+  - Set the Startup Project to `QuartzSandbox`.
+  - At this point you should be able to run, since the project should have already been
+    built in step 2. above. You can always build the traditional way with Visual Studio.
+  - And voila, all done. Now you should be able to run the project!
 
-  1. `mkdir build && cd build`
-  2. `cmake ..`
-  3. `make`
+### Linux, Mac OS X, MSYS
+ 
+  - Navigate to the `Build/` folder and run `./QuartzSandbox` to run the executable.
```

Comment:
So they need to navigate to Build/<project>/ then run ./<project>
In this case CD Build/QuatzSandbox/ then ./QuartzSandbox

Reply:
Ah damn I did wonder if I had got this the right way round :D Trying to write linux instructions from windows was always going to end badly.

I'll fix this now